students/todo.md

```diff
@@ -62,15 +62,19 @@ Now there's only one thing left to say: Use all the channels there are and ask a
 
 Most of the aspects mentioned here need to be done at the beginning of the program. Follow these steps and you'll have a great setup for all the activities.
 
-<h3>Github Private Repo</h3>      
-A separate Github repo is the best place for your team to organize the issues to work on. We will provide a private repo for each team so you can safely get on with your work.
+<h3>RGSoC-Orga Slack Team</h3>
+All the members of the team will be invited to our Slack RGsoC-Orga Team. This is currently our favourite channel of communication and we set up several channels with different purposes so that you won't miss any important information. You can also talk with each other via private message.
```

Comment:
"All the members of the team" is a bit vague, maybe replace with "All RGSoC participants" or "All program participants".